build.sbt

```diff
@@ -177,6 +178,29 @@ lazy val python =
       publish / skip := true
     )
 
+lazy val docs =
+  (project in file("docs"))
+    .dependsOn(core % "test->test")
+    .settings(
+      test in Test := {
+        // Pass the test classpath to pyspark so that we run the same bits as the Scala tests
+        val classpath = (fullClasspath in Test)
```

Comment:
Can we factor this into `commonSettings` (or other shared value)? Since tis' used here and in `python`.